pixhawk-companion-computer.md

```diff
@@ -38,3 +38,46 @@ The safe bet is to use an FTDI Chip USB-to-serial adapter board and the wiring b
 |4         | CTS (in) |6       | FTDI RTS (green) (out) |
 |5         | RTS (out)|2       | FTDI CTS (brown) (in) |
 |6         | GND     | 1       | FTDI GND (black)   |
+
+## Software setup on Linux
+
+On Linux the default name of a USB FTDI would be like `\dev\ttyUSB0`. If you have a second FTDI linked on the USB or an Arduino, it will registered as `\dev\ttyUSB1`. To avoid the confusion between the first plugged and the second plugged, it is a good maneer to create a symlink it by the . To do it, you can use UDEV rules based on the Vendor and Product ID of the USB device.
+
+Using `lsusb` we can get the vendor and product IDs.
+
+```sh
+$ lsusb
+
+Bus 006 Device 002: ID 0bda:8153 Realtek Semiconductor Corp.
+Bus 006 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 002: ID 05e3:0616 Genesys Logic, Inc.
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 004: ID 2341:0042 Arduino SA Mega 2560 R3 (CDC ACM)
+Bus 003 Device 005: ID 26ac:0011
+Bus 003 Device 002: ID 05e3:0610 Genesys Logic, Inc. 4-port hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 002 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
+Bus 001 Device 002: ID 0bda:8176 Realtek Semiconductor Corp. RTL8188CUS 802.11n WLAN Adapter
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+
+The Arduino is `Bus 003 Device 004: ID 2341:0042 Arduino SA Mega 2560 R3 (CDC ACM)`
+
+The Pixhawk is `Bus 003 Device 005: ID 26ac:0011`
+
+Therefore, we can create a new UDEV rule in a file called `/etc/udev/rules.d/99-pixhawk.rules` with the following content, changing the idVendor and idProduct to yours.
+
+```sh
+SUBSYSTEM=="tty", ATTRS{idVendor}=="2341", ATTRS{idProduct}=="0042", SYMLINK+="ttyLasers"
```

Comment:
`ttyLasers` seems a bit too specific to your setup. Can you use a more generic name? `ttyArduino` maybe?